### ISSUE TITLE
Preserved parts subfield 852$q not in holding configuration

### ISSUE THREAD

**Issue by HirschSt** [rism-ch/muscat#448]
This is missing in the holding marc conf; it's also not in the holding form template, so there is the risk that values in this field get lost after saving. 
Value is in the export, eg. 00000990052750:

```
<datafield tag="852" ind1="1" ind2="0">
    <subfield code="a">PL-Kj</subfield>
    <subfield code="c">Mus.ant.pract. P 1265</subfield>
    <subfield code="q">kpl.: prima vox, altera vox, tertia vox</subfield>
    <subfield code="x">30002079</subfield>
    <subfield code="z">BICINIA vnd TRICINIA | Michaëlis Praetorij, C. | Darinnen | Die meiste Psalmen vnd | Geistliche/ in Kirchen vnd Häusern gebreuchliche | Lieder mit 2. vnnd 3. Stimmen/ auff Muteten, Madrigali=|sche vnnd sonsten noch eine andere/ vom Autore erst er=|fundene Art (wie davon in der Nota Autoris ad | Lectorem Musicum mehrer Bericht | zubefinden) gesetzet | seind. | PRIMA VOX. | 1611. | Vom Autore verlegt/ vnd nach nach vollene=|tem Druck anderweit corrigirt. | Bey Michel Hering/ in Hamburg | zufinden. [at the end of the volume:] Wolffenbüttel | Gedruckt in Fürstlicher Druckerey/ in | Verlegung des Autoris.</subfield>
    <subfield code="z">Ex libris: "Ex Bibliotheca Poelchaviana"; "Liber Biblioth. Academ. Francof."</subfield>
  </datafield>
```

**Comment by xhero** [comment]
Can you add this directly to -devel ? 


Inviato dal mio dispositivo Samsung

-------- Messaggio originale --------
Da: Stephan Hirsch <notifications@github.com> 
Data: 11/11/2016  15:31  (GMT+01:00) 
A: rism-ch/muscat <muscat@noreply.github.com> 
Oggetto: [rism-ch/muscat] Preserved parts subfield 852$q not in holding
  configuration (#448) 

This is missing in the holding marc conf; it's also not in the holding form template, so there is the risk that values in this field get lost after saving.

Value is in the export, eg. 00000990052750:
<datafield tag="852" ind1="1" ind2="0">
    <subfield code="a">PL-Kj</subfield>
    <subfield code="c">Mus.ant.pract. P 1265</subfield>
    <subfield code="q">kpl.: prima vox, altera vox, tertia vox</subfield>
    <subfield code="x">30002079</subfield>
    <subfield code="z">BICINIA vnd TRICINIA | Michaëlis Praetorij, C. | Darinnen | Die meiste Psalmen vnd | Geistliche/ in Kirchen vnd Häusern gebreuchliche | Lieder mit 2. vnnd 3. Stimmen/ auff Muteten, Madrigali=|sche vnnd sonsten noch eine andere/ vom Autore erst er=|fundene Art (wie davon in der Nota Autoris ad | Lectorem Musicum mehrer Bericht | zubefinden) gesetzet | seind. | PRIMA VOX. | 1611. | Vom Autore verlegt/ vnd nach nach vollene=|tem Druck anderweit corrigirt. | Bey Michel Hering/ in Hamburg | zufinden. [at the end of the volume:] Wolffenbüttel | Gedruckt in Fürstlicher Druckerey/ in | Verlegung des Autoris.</subfield>
    <subfield code="z">Ex libris: "Ex Bibliotheca Poelchaviana"; "Liber Biblioth. Academ. Francof."</subfield>
  </datafield>


—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/rism-ch/muscat","title":"rism-ch/muscat","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/rism-ch/muscat"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Preserved parts subfield 852$q not in holding configuration (#448)"}],"action":{"name":"View Issue","url":"https://github.com/rism-ch/muscat/issues/448"}}}

**Comment by HirschSt** [comment]
Yep, fixed with f553468
